README.md

```diff
@@ -290,7 +290,7 @@ The following variables are available in the template:
 - `webpackConfig`: the webpack configuration that was used for this compilation. This
   can be used, for example, to get the `publicPath` (`webpackConfig.output.publicPath`).
 
-- `compilation`: the webpack [compilation](https://webpack.js.org/api/compilation/) object.
+- `compilation`: the webpack [compilation](https://webpack.js.org/api/compilation-hooks/) object.
```

Comment:
![image](https://user-images.githubusercontent.com/10627668/61639467-807a7a80-acd6-11e9-949e-9ed89ad751bd.png)

The webpack docs link to `compilation-hooks` when clicked compilation button.

Reply:
Yes but during render you don't want to use hooks but rather output data from the compilation like file size or file names